src/environmentd/src/http.rs

```diff
@@ -476,17 +509,21 @@ impl AuthedClient {
         active_connection_count: SharedConnectionCounter,
         session_config: F,
         options: BTreeMap<String, String>,
+        peer_addr: IpAddr,
```

Comment:
Can you add this new parameter next to the `user` parameter? Feels like it morally fits better there.